packages/flutter/lib/src/material/input_decorator.dart

```diff
@@ -709,6 +711,8 @@ class _RenderDecoration extends RenderBox {
     markNeedsSemanticsUpdate();
   }
 
+  bool expands;
```

Comment:
@HansMuller Can you point me in the right direction for how to test this?  Our existing tests modify parameters by creating new widgets, but that won't test changing the param:

```dart
    await tester.pumpWidget(expandedTextFieldBuilder(expands: false));
    // Expect height to be small
    await tester.pumpWidget(expandedTextFieldBuilder(expands: true));
    // Expect height to be bigger.
    // This passes, but it would pass with or without the fix you suggested.
```

Also, I can't directly access this class because it's private, so I wouldn't be able to more directly test it.